recipes/zookeeper-client-c/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+class ZookeeperClientCConan(ConanFile):
+    name = "zookeeper-client-c"
+    description = "ZooKeeper is a centralized service for maintaining configuration information, naming, providing distributed synchronization, and providing group services."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://zookeeper.apache.org/"
+    topics = ("zookeeper", "client")
+    settings = "os", "arch", "compiler", "build_type"
+    package_type = "static-library"
+    options = {
+        "fPIC": [True, False],
+        "with_cyrus_sasl": [True, False],
+        "with_openssl": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "with_cyrus_sasl": False,
+        "with_openssl": False,
+    }
+    short_paths = True
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.with_cyrus_sasl:
+            self.requires("cyrus-sasl/2.1.27")
+        if self.options.with_openssl:
+            self.requires("openssl/[>=1.1 <4]")
+
+    def build_requirements(self):
+        self.tool_requires("zulu-openjdk/11.0.15")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["WANT_CPPUNIT"] = False
+        tc.variables["WITH_CYRUS_SASL"] = "ON" if self.options.with_cyrus_sasl else "OFF"
+        tc.variables["WITH_OPENSSL"] = "ON" if self.options.with_openssl else "OFF"
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def build(self):
+        apply_conandata_patches(self)
+
+        # We have to install maven to generate jute files which are required by zookeeper-client
+        get(self,
+            "https://dlcdn.apache.org/maven/maven-3/3.9.2/binaries/apache-maven-3.9.2-bin.tar.gz",
+            destination=os.path.join(self.source_folder, "maven"),
+            strip_root=True
+        )
```

Comment:
Can you make `maven` a `tool_requires` with its corresponding new recipe? It could work like the new cmake one which only downloads the precompiled oficial binaries, and this might be useful for other uses that also have a similar need. Let me know if you need any help with that to move it forward :)

Reply:
@RubenRBS 
I agree with you.
I try to make maven recipe.

Reply:
@RubenRBS 
I created maven recipe.
Please review it.
https://github.com/conan-io/conan-center-index/pull/17868

Reply:
@RubenRBS 
Now maven recipe has been merged.
I update this recipe with maven.
Please review it again.

Reply:
Thanks a lot, will do tomorrow, sorry it took so long to review Maven :)